components/climate/thermostat.rst

```diff
@@ -315,6 +315,95 @@ Set Point Options/Behavior
 **If configured, at least one of** ``default_target_temperature_low`` **and** ``default_target_temperature_high``
 **must be specified in the away mode configuration.**
 
+Presets
+*******
+
+Presets allow you to preconfigure different modes for your thermostat. These will show up in the HomeAssistant
```

Comment:
"Home Assistant" is two words. 🙂

Reply:
Cheers!

Not sure if you're aware but you can also leave suggestions by leaving a line-specific comment with a multi-line code block with the language of "suggestion" (ie. ```suggestion...). Then the Github UI let's the owner just click accept the suggestions and commit them. Not suitable for all issues on a PR, but great for typos!